bundles/org.openhab.binding.nuvo/doc/NuvoMenu.xsd

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Is this referenced somewhere?

Reply:
No it is not. I put this here for a future developer should they wish to regenerate the DTO from the XSD. I just added a note in the file that provides the _'xjc'_ command with all necessary arguments.